server/api/models/question.js

```diff
@@ -2,9 +2,16 @@ const mongoose = require("mongoose");
 
 // Question Schema
 const QuestionSchema = mongoose.Schema({
-  question: { type: String },
-  answer: { type: String },
-  options: [{ type: String }]
+  title: String,
+  options: [
+    {
+      value: String,
+      isCorrect: Boolean
+    }
+  ],
+  created_at: Date,
+  updated_at: Date,
+  is_deleted: Boolean
```

Comment:
Let's move `is_deleted` above created and updated at and what do you think about renaming `is_deleted` to `status`?

Reply:
You know what, let's keep the `is_deleted` but give these things a default. Let's do that and merge this asap :)

Reply:
Changes done, default values assigned.